ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -323,6 +323,11 @@ final class SandboxServer(
           authService.getClass.getSimpleName,
           config.seeding.fold("no")(_.toString.toLowerCase),
         )
+        if (config.scenario.nonEmpty) {
+          logger.withoutContext.warn(
+            """Initializing a ledger with scenarios is deprecated and will be removed in the future. You are advised to use DAML Script instead.
```

Comment:
Do you want to add a sentence along the lines of, "Test scenarios in DAML Studio will continue to work as expected."?

Reply:
Thanks. Fixed in 6cb255c.